### PR TITLE
Add doc how to use customized porperties file

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR adds a document about how to use a customized properties files.
This document is related to the following new features.

https://github.com/scalar-labs/helm-charts/pull/96
https://github.com/scalar-labs/helm-charts/pull/102
https://github.com/scalar-labs/helm-charts/pull/103

Please take a look!

### CODE REVIEW

**Comment by kota2and3kan** [comment]
I will update this PR based on the #96 after it will be merged.

**Comment by kota2and3kan** [comment]
I updated the document based on the new features of the following PRs.

https://github.com/scalar-labs/helm-charts/pull/96
https://github.com/scalar-labs/helm-charts/pull/102
https://github.com/scalar-labs/helm-charts/pull/103

Please take a look!

File: docs/how-to-use-customized-configuration.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    * Scalar DL Auditor
+      ```yaml
+      auditor:
+        auditorProperties: |
+          ...
+          scalar.dl.auditor.cert_path=/keys/certificate
+          scalar.dl.auditor.private_key_path=/keys/private-key
+      ```
+
+In this example, you need to mount a **private-key** and a **certificate** file under the `/key` directory in the container. And, you need to mount files named `private-key` and `certificate`. You can use `extraVolumes` and `extraVolumeMounts` to mount these files.
```
Comment (ymorimo) [inline docs/how-to-use-customized-configuration.md:184]: Probably the `/keys` is a correct one?

```suggestion
In this example, you need to mount a **private-key** and a **certificate** file under the `/keys` directory in the container. And, you need to mount files named `private-key` and `certificate`. You can use `extraVolumes` and `extraVolumeMounts` to mount these files.
```

File: docs/how-to-use-customized-configuration.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+         ```
+       * Scalar DL Auditor
+         ```console
+         kubectl create secret generic auditor-key-secret \
+           --from-file=private-key=./auditor-key.pem \
+           --from-file=certificate=./auditor-cert.pem
+         ```
+
+1. Deploy Scalar products with the above custom values file.  
+   After deploying Scalar products, key and certificate files are mounted under the `key` directory as follows.
```
Comment (ymorimo) [inline docs/how-to-use-customized-configuration.md:229]: Ditto.

```suggestion
   After deploying Scalar products, key and certificate files are mounted under the `/keys` directory as follows.
```

File: docs/how-to-use-customized-configuration.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+   * Example
+       * Scalar DL Ledger
+         ```console
+         $ ls -l /keys/
+         total 0
+         lrwxrwxrwx 1 root root 18 Jun 27 03:12 private-key -> ..data/private-key
```
Comment (ymorimo) [inline docs/how-to-use-customized-configuration.md:235]: Is `..data/private-key` displayed as it is or does the `..` represents an abbreviation? If an abbreviation, I think something like `.../data/private-key` or `/path/to/data/private-key` is more appropriate.

Comment (ymorimo) [inline docs/how-to-use-customized-configuration.md:235]: Is `..data/private-key` displayed as it is or does the `..` represents an abbreviation? If an abbreviation, I think something like `.../data/private-key` or `/path/to/data/private-key` is more appropriate.

**Review by ymorimo — COMMENTED** [review]
Overall looking good. I left some minor comments. Please take a look! Sorry for the late review.

File: docs/how-to-use-customized-configuration.md (hunk 1 above)
Comment (kota2and3kan) [reply docs/how-to-use-customized-configuration.md]: You are right. It's typo...
Thank you for catching it!

**Comment by kota2and3kan** [comment]
@ymorimo  
Thank you for your review!

I fixed some pointed out things and left some comment.
Please take a look again.

File: docs/how-to-use-customized-configuration.md (hunk 3 above)
Comment (ymorimo) [reply docs/how-to-use-customized-configuration.md:235]: At a first look `..` seemed somewhat strange to me, but I understand that it is part of a real (hidden) file name. Thank you!

**Review by ymorimo — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**